ppb/features/default_sprites.py

```diff
@@ -0,0 +1,43 @@
+import ppb
+import math
+
+
+class TargetSprite(ppb.sprites.BaseSprite):
```

Comment:
```suggestion
class TargetSprite(ppb.Sprite):
```

Just to make sure all of the expected features are available.